packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -3580,7 +3594,7 @@ export interface IDetailsRowState {
     // (undocumented)
     isDropping?: boolean;
     // (undocumented)
-    selectionState?: IDetailsRowSelectionState;
+    selectionState: IDetailsRowSelectionState;

```

Comment:
This might introduce compiler errors for folks. Doubtful that folks are actually using it tho...

Reply:
I don't think we usually consider state to be part of the public API surface (though if that's the case we really ought to get it out of this file...)